src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
> Funny to read when the one person who fixes bugs in OPAM (Louis Gesbert) is actually an OCamlPro employee.

You are mixing two things that are completely unrelated. Louis is certainly not the only person fixing bugs with the opam repository, I'm not even sure he is an opam repository maintainer. 

> It takes me more time to discuss on this forum everything I would like to do than implementing them and starting them on OCamlPro's servers.

This is not my problem. You may want to reflect on the way you interact with other people and the community or on the solutions you propose that always try to tie in your company infrastructure in some way.

> If your goal is to demotivate outside contributions to OPAM and its repository, you are quite efficient !

My goal is to have a sustainable environment. I don't see an environment partly in control of OCamlPro or any other company as being a sustainable environment. The environment should be in control of the community which is what opam.ocaml.org is for, as such I don't feel we should have URIs that point to OCamlPro's infrastructure in the opam source itself.